Ethereal Artefacts/Ethereal Artefacts/Views/LoginView.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  LoginView.swift
+//  Ethereal Artefacts
+//
+//  Created by Albert Khurshudyan on 3.05.23.
+//
+
+import SwiftUI
+
+struct LoginView: View {
+    @State private var userEmail: String = ""
+    @State private var password: String = ""
+    @State private var isEmailValid: Bool = true
+    @State private var showPassword: Bool = false
+    @State private var loginError: String? = nil
+    @State private var isLoggedIn: Bool = false
+    
+    let validUserEmail = "user@example.com"
+    let validPassword = "112233"
+    
+    var body: some View {
+        NavigationView {
+            ZStack {
```

Comment:
This is the old way of navigation the new one is with NavigationStack

Reply:
Replaced.